vic/drivers/shared_image/src/check_domain_info.c

```diff
@@ -58,7 +58,7 @@ compare_ncdomain_with_global_domain(char *ncfile)
     // loop over all grid cells and check that the two domains are identical
     for (i = 0; i < global_domain.ncells_total; i++) {
         // mask matches
-        if (ncfile_domain.locations[i].run > global_domain.locations[i].run) {
+        if (ncfile_domain.locations[i].run < global_domain.locations[i].run) {
```

Comment:
I'm not sure what we want to do here.  Both the domain and parameter files have a `mask` variable. This variable currently populates the `locations.run` attribute.  The question is, do we want to be able to turn on and off which grid cells are run?  And if so, which file and variable should this be done in.  

The simplest solution is to say no and assert, as @bartnijssen suggested, that the masks match exactly.